ortb/default.go

```diff
@@ -66,6 +88,16 @@ func setDefaultsTargeting(targeting *openrtb_ext.ExtRequestTargeting) bool {
 	return modified
 }
 
+func adjustDefaultsPriceGranularity(priceGranularity *openrtb_ext.PriceGranularity) (*openrtb_ext.PriceGranularity, bool) {
+	if priceGranularity == nil || len(priceGranularity.Ranges) == 0 {
```

Comment:
Consider adding this condition in the `setDefaultsPriceGranularity` function itself at the very start. Something like:
```
func setDefaultsPriceGranularity(pg *openrtb_ext.PriceGranularity) (*openrtb_ext.PriceGranularity, bool) {
	if pg == nil || len(pg.Ranges) == 0 {
		pg = ptrutil.ToPtr(openrtb_ext.NewPriceGranularityDefault())
		return pg, true
	}
	modified := false
	if pg.Precision == nil {
		pg.Precision = ptrutil.ToPtr(DefaultPriceGranularityPrecision)
		modified = true
	}

	if setDefaultsPriceGranularityRange(pg.Ranges) {
		modified = true
	}

	return pg, modified
}
```

The reason I am saying this is because having two functions with names `adjustDefaultsPriceGranularity` and `setDefaultsPriceGranularity` doesn't clearly describe the difference between what they are doing and I think we can easily accommodate this behavior in the `setDefaultsPriceGranularity` function itself

Reply:
Good point, modified